IngresseSDK/Services/Cashless/CashlessURLRequest.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CashlessURLRequest.swift
+//  IngresseSDK
+//
+//  Created by Fernando Ferreira on 26/04/20.
```

Comment:
Geralmente a gente deixa assim no cabeçalho:

```suggestion
//
//  Copyright © 2020 ingresse. All rights reserved.
//
```